release.md

```diff
@@ -0,0 +1,92 @@
+- **major**: backwards-breaking changes
+- **minor**: functionality added in a backwards compatible manner
+- **patch**: backwards compatible bug fixes
+
+## Pre-Release Tasks
+
+### Start Release Ticket
+
+Move the JIRA ticket for the release to the "In Progress" state.
+Ensure that its fixVersion matches the version being released.
+
+### Complete the Release in JIRA
+
+Go to the [SQL releases page](https://jira.mongodb.org/projects/SQL?selectedItem=com.atlassian.jira.jira-projects-plugin%3Arelease-page&status=unreleased), and ensure that all the tickets in the fixVersion to be released are closed.
+Ensure that all the tickets have the correct type. Take this opportunity to edit ticket titles if they can be made more descriptive.
+The ticket titles will be published in the changelog.
+
+If you are releasing a patch version but a ticket needs a minor bump, update the fixVersion to be a minor version bump.
+If you are releasing a patch or minor version but a ticket needs a major bump, stop the release process immediately.
+
+The only uncompleted ticket in the release should be the release ticket.
+If there are any remaining tickets that will not be included in this release, remove the fixVersion and assign them a new one if appropriate.
+
+Close the release on JIRA, adding the current date (you may need to ask the SQL project manager to do this).
+
+### Ensure Evergreen Passing
+
+Ensure that the build you are releasing is passing the tests on the
+[evergreen waterfall](https://evergreen.mongodb.com/waterfall/mongo-powerbi-connector).
+
+## Release Tasks
+
+### Ensure master up to date
+
+Ensure you have the `main` branch checked out, and that you have pulled the latest
+commit from [mongodb/mongo-powerbi-connector](https://github.com/mongodb/mongo-powerbi-connector).
+
+### Create the tag and push
+
+Create an annotated tag and push it:
+
+```sh
+git tag -a -m X.Y.Z vX.Y.Z
+git push --tags
+```
+
+This should trigger an Evergreen version that can be viewed on the
+[mongo-powerbi-connector waterfall](https://evergreen.mongodb.com/waterfall/mongo-powerbi-connector).
+
+If it does not, you may have to ask the project manager to give you the right permissions to do so.
+Make sure to run the 'release' task, if it is not run automatically.
+
+### Set Evergreen Priorities
+
+Some evergreen variants may have a long schedule queue.
+To speed up release tasks, you can set the task priority for any variant to 101 for release candidates and 200 for actual releases.
+If you do not have permissions to set priority above 100, ask the project manager to set the
+priority.
+
+## Post-Release Tasks
+
+### Wait for evergreen
+
+Wait for the evergreen version to finish, and ensure that the release task completes successfully.
+
+### Verify release artifacts
+
+Check that the released file is available at the URL:
+- Windows
+`https://translators-connectors-releases.s3.us-east-1.amazonaws.com/mongosql-odbc-driver/windows/${release_version}/mongoodbc.dll`
```

Comment:
This can be updated with the powerbi location: mongo-powerbi-connector/MongoDBAtlasODBC-${RELEASE_VERSION}.pqx

Reply:
Thanks, missed this one!